src/test/resources/infochar.vcf

```diff
@@ -121,4 +121,4 @@
 ##contig=<ID=GL000192.1,length=547496>
 ##contig=<ID=NC_007605,length=171823>
 #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	C1046::HG02024	C1046::HG02025
-20	10019093	rs575534	A	G	1.68684e+06	PASS	dbNSFP_PROVEAN_pred=.,N,N	GT:AD:DP:GQ:PL	0/0:30,0:30:72:0,72,1080	0/1:49,45:94:99:1352,0,1480
+20	10019093	rs575534	A	G	1.68684e+06	PASS	dbNSFP_PROVEAN_pred=N,N,N	GT:AD:DP:GQ:PL	0/0:30,0:30:72:0,72,1080	0/1:49,45:94:99:1352,0,1480
```

Comment:
Yeah, so this is going to be a problem.  I think we should always allow arrays in the INFO field to have missing elements, and the flag should only control the requiredness of arrays in the genotype fields.

I just did an example and the current VCF parser treats the entire PL field as missing if any element is missing: `0/1:49,45:94:99:.,0,1480` parses as `0/1:49,45:94:99:.`.